index.html

```diff
@@ -9,6 +9,7 @@
     />
     <title>Martin Karugaba</title>
     <link rel="stylesheet" href="./dist/output.css" />
+    <link rel="stylesheet" href="style.css" />
```

Comment:
[OPTIONAL] 🤓 Seems like you wanted to use this `style.css` file. However, right now it doesn't exist and is throwing a 404.  I strongly recommend you either remove the link or add the file.